src/views/Dashboard.vue

```diff
@@ -61,7 +61,57 @@
                         <p>Summer 2023</p>
                     </div>
                 </div>
-                <!-- <div class="user-experience"></div> -->
+                <div id="user-experience">
+                    <div id="xp-progress-bar">
+                        <div id="current-progress"></div>
+                    </div>
+                    <p style="align-self: flex-start; margin-left: 0.5rem;">Level 27 <strong>1.2K / 1.5K Experience</strong></p>
+
+                    <div id="rewards-basic" class="rewards-container">
```

Comment:
We can use `v-for` for this in the future

https://vuejs.org/guide/essentials/list.html